src/bootstrapper/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+set(BOOTSTRAPPER_TARGET_NAME bootstrapper)
+
+configure_file(bootstrapper.rc.in resources/bootstrapper.rc)
+
+file(GLOB BOOTSTRAPPER_HEADERS LIST_DIRECTORIES=false *.h)
+file(GLOB BOOTSTRAPPER_SOURCES LIST_DIRECTORIES=false *.cpp)
+file(GLOB BOOTSTRAPPER_RESOURCES LIST_DIRECTORIES=false ${CMAKE_CURRENT_BINARY_DIR}/resources/*.rc)
+
+add_executable(${BOOTSTRAPPER_TARGET_NAME}
+	WIN32
+	${BOOTSTRAPPER_HEADERS}
+	${BOOTSTRAPPER_SOURCES}
+	${BOOTSTRAPPER_RESOURCES}
+)
+
+target_compile_features(${BOOTSTRAPPER_TARGET_NAME}
+	PRIVATE
+		cxx_std_17

```

Comment:
This should be configured by `cmake`/`CMakeToolchain`. We should investigate, it is it not.